src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Bugzilla/Bugzilla39331.cs

```diff
@@ -1,4 +1,5 @@
-﻿using NUnit.Framework;
+﻿#if TEST_FAILS_ON_ANDROID && TEST_FAILS_ON_CATALYST && TEST_FAILS_ON_IOS && TEST_FAILS_ON_WINDOWS //Issue reproduced and logged: https://github.com/dotnet/maui/issues/26019.
```

Comment:
The issue was a duplicated and has been closed, could you update the comment to replace it by https://github.com/dotnet/maui/issues/26026?

Reply:
Replaced the issue link and committed the changes, kindly review and share your feedback.